tardis/workflows/util.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from astropy import constants as const
+
+
+def get_tau_integ(plasma, simulation_state, bin_size=10):
+    """Estimate the integrated mean optical depth at each velocity bin
+
+    Parameters
+    ----------
+    plasma : tardis.plasma.BasePlasma
+        The tardis legacy plasma
+    simulation_state : tardis.model.base.SimulationState
+        the current simulation state
+    bin_size : int, optional.  Default : 10
+        bin size for the aggregation of line opacities
+
+    Returns
+    -------
+    dict
+        rosseland : np.ndarray
+            Roassland Mean Optical Depth
+        planck : np.ndarray
+            Planck Mean Optical Depth
+    """
+    index = plasma.atomic_data.lines.nu.index
+    taus = plasma.tau_sobolevs.loc[index]
+    freqs = plasma.atomic_data.lines.nu.values
+    order = np.argsort(freqs)
+    freqs = freqs[order]
+    taus = plasma.tau_sobolevs.values[order]
+
+    extra = bin_size - len(freqs) % bin_size
```

Comment:
I can't tell what the point of this bit of code is. Could there be more informative names, or at least a comment on why you need to handle the extra frequency bins like this?